### ISSUE TITLE
/msgtoggle

### ISSUE THREAD

**Issue by notchface72** [TotalFreedom/TotalFreedomMod#663]
put as a command suggestion per 
![image](https://cloud.githubusercontent.com/assets/10501056/7786240/014de1a0-0174-11e5-8ee9-35a28374ac83.png)
this command would work the same as tptoggle, except it would disable your ability to recieve and send messages, whispers and tells rather than disabling the ability from players to tp to you. stampy100 has given me this idea.

**Comment by JeromSar** [comment]
@notchface72 I don't think this is a good idea, it's always important to receive /msgs from admin. Ignorance is never bliss.

**Comment by Wild1145** [comment]
@JeromSar I think it would be a good thing to have, but just have it so that admin messages bypass that filter? 

I also dont often see (Ever seen) an admin send a player a message for anything important, its normally just something like /say <player> Stop doing this or stop doing that.
